generate/convert.go

```diff
@@ -439,6 +439,11 @@ func (g *generator) convertDefinition(
 				return nil, err
 			}
 
+			if options.GetPointer() || (!field.Type.NonNull && g.Config.Optional == "pointer") {
```

Comment:
I wonder if this should apply to non-input structures as well..?

Reply:
omitempty is meaningless for outputs (since it's meaningless on unmarshal)